lib/templates-sync.php

```diff
@@ -18,20 +18,21 @@
  * @param string $content   Template content.
  */
 function _gutenberg_create_auto_draft_for_template( $post_type, $slug, $theme, $content ) {
-	// We check if an auto-draft was already created,
+	// We check if a post for this template was already created,
 	// before running the REST API calls
-	// because the site editor needs an existing auto-draft
+	// because the site editor needs an existing post
 	// for each theme template part to work properly.
 	$template_query = new WP_Query(
 		array(
 			'post_type'      => $post_type,
-			'post_status'    => array( 'publish', 'auto-draft' ),
+			'post_status'    => array( 'publish' ),
```

Comment:
Similar to before, I think publish status is default, so we don't have to add it